source/tests/basic_timing/time_stats.cpp

```diff
@@ -0,0 +1,185 @@
+/*
+ * Copyright (c) 2017- Hourglass Resurrection Team
+ * Hourglass Resurrection is licensed under GPL v2.
+ * Refer to the file COPYING.txt in the project root.
+ */
+
+#include "stdafx.h"
+#include "logger.h"
+#include "time_stats.h"
+
+namespace TimeStats
+{
+    namespace
+    {
+        unsigned gs_frame_counter = 0;
```

Comment:
Please specify a type. I know `unsigned` in itself is a type, but please specify also `int` or whatever is appropriate.